README.md

```diff
@@ -41,16 +41,30 @@ client.authorize(credentials)
 	});
 ```
 
-#### #listWraps([queryParams])
-Returns a list of all Wraps for an account. An optional `queryParams` argument can include filter and sort request parameters. These parameters can be reviewed in the wrapi-rails apidocs.
+#### #listWraps([search])
```

Comment:
I guess this is coming down the line but it would be awesome to explicitly list all the possible parameters this can take. Alternatively, to prevent duplication, maybe link to the actual wrapi-rails endpoint that this call uses (i.e. https://wrapi.wrap.co/apidocs#!/wraps/Api_Wraps_search_get_0)?